Tests/test_imagefont.py

```diff
@@ -462,15 +461,15 @@ def test_unicode_pilfont(self):
         # issue #2826
         font = ImageFont.load_default()
         with self.assertRaises(UnicodeEncodeError):
-            font.getsize(u"’")
+            font.getsize("’")
 
     @unittest.skipIf(
         sys.version.startswith("2") or hasattr(sys, "pypy_translation_info"),
         "requires CPython 3.3+",
     )
```

Comment:
Can drop the `sys.version.startswith("2")` check:

```diff
diff --git a/Tests/test_imagefont.py b/Tests/test_imagefont.py
index 6daac701..d2e24836 100644
--- a/Tests/test_imagefont.py
+++ b/Tests/test_imagefont.py
@@ -463,10 +463,7 @@ class TestImageFont(PillowTestCase):
         with self.assertRaises(UnicodeEncodeError):
             font.getsize("’")
 
-    @unittest.skipIf(
-        sys.version.startswith("2") or hasattr(sys, "pypy_translation_info"),
-        "requires CPython 3.3+",
-    )
+    @unittest.skipIf(hasattr(sys, "pypy_translation_info"), "requires CPython")
     def test_unicode_extended(self):
         # issue #3777
         text = "A\u278A\U0001F12B"
```

Reply:
Thanks, removed along with some other stuff in https://github.com/python-pillow/Pillow/pull/4109/commits/324e6e4973a4737a12e2bf2bebf6d68efe44be12.